tests/www/views/test_views_connection.py

```diff
@@ -62,11 +62,8 @@ def test_invalid_connection_id_trailing_blanks(admin_client, session):
     invalid_conn_id = "conn_id_with_trailing_blanks   "
     invalid_connection = {**CONNECTION, "conn_id": invalid_conn_id}
     resp = admin_client.post("/connection/add", data=invalid_connection, follow_redirects=True)
-    check_content_in_response(
-        f"The key '{invalid_conn_id}' has to be made of alphanumeric characters, "
-        + "dashes, dots and underscores exclusively",
-        resp,
-    )
+    # all the whitespaces get stripped off
+    check_content_in_response("Added Row", resp)
```

Comment:
You dont seem to check the blanks have actually been stripped? Also, the test name needs to be updated

Reply:
Sure I will make the changes for test name.
@vincbeck I am unable to figure out how to check if the blanks have been stripped off using unit tests..
Any hint would be really nice!